baseplate/clients/redis_cluster.py

```diff
@@ -347,7 +378,7 @@ class ClusterRedisContextFactory(ContextFactory):
     :param connection_pool: A connection pool.
     """
 
-    PROM_PREFIX = "bp_redis_cluster_pool"
```

Comment:
since `PROM_PREFIX = "redis_cluster_client"` and then `PROM_PREFIX = f"{PROM_PREFIX}_pool"`, we get `redis_cluster_client_pool`, so _I think_ my diff matches your PR changes. lmk if I missed something